rules/Multifactor-Google-Authenticator-Do-Not-Rename.js

```diff
@@ -1,30 +1,36 @@
 function (user, context, callback) {
 
-  var mfa_enabled_connection = [
-    'github',
-    'google-oauth2'
-  ].indexOf(context.connection) !== -1;
+    var mfa_enabled_connection = [
+        'github',
+        'google-oauth2'
+    ].indexOf(context.connection) !== -1;
 
-  var user_with_mfa = user.app_metadata && user.app_metadata.use_mfa;
+    // Exclude the following clients from mfa
+    var clients = [
```

Comment:
I'd rename this to something more descriptive, like `mfa_enabled_client` - the following test would then read more naturally:
```
if (
        user_with_mfa &&
        mfa_enabled_connection &&
        mfa_enabled_client
    ) {
```